addons/metadata.integral.artists/lib/musicbrainz.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: UTF-8 -*-
+def musicbrainz_artistfind(data):
+    artists = []
+    for item in data.get('artists',[]):
+        artistdata = {}
+        artistdata['artist'] = item['name']
+        artistdata['thumb'] = ''
+        artistdata['genre'] = ''
+        artistdata['born'] = ''
+        artistdata['mbid'] = item['id']
+        if item.get('score',1):
+            artistdata['relevance'] = str(item['score'] / 100.00)
+        artists.append(artistdata)
+    return artists
+
+def musicbrainz_artistdetails(data):
+    artistdata = {}
+    artistdata['artist'] = data['name']
+    artistdata['mbartistid'] = data['id']
+    if data.get('life-span','') and data.get('type',''):
```

Comment:
Can "type", "gender", and "disambiguation" from the Musicbrainz results be set in the artist data too. Also needs adding to wherever artistdata is passed back to CArtist

Reply:
ok, i've added 'type', 'genre' & 'disambiguation'.

the only other source we use for searching is discogs.com, it does not provide this info